server/plugin/api.go

```diff
@@ -495,7 +495,7 @@ func (p *Plugin) handleDriveWatchNotifications(c *Context, w http.ResponseWriter
 
 	activitySrv, err := driveactivity.NewService(context.Background(), option.WithTokenSource(conf.TokenSource(context.Background(), authToken)))
 	if err != nil {
-		p.API.LogError("Failed to fetch changes", "err", err)
+		p.API.LogError("Failed to fetch google drive changes", "err", err)
```

Comment:
Let's be consistent with the captialization for "google drive" so that searching logs is easier.